src/dmd/dtoh.d

```diff
@@ -1116,7 +1116,7 @@ public:
         checkbuf.level++;
         const sn = sd.ident.toChars();
         const sz = sd.size(Loc.initial);
-        checkbuf.printf("assert(sizeof(%s) == %d);", sn, sz);
+        checkbuf.printf("assert(sizeof(%s) == %lu);", sn, sz);
```

Comment:
`sz` is a `uint64_t`, so this format should use `llu` instead of `lu`.

Reply:
Done.